layers/meta-balena-jetson/recipes-bsp/u-boot/u-boot-tegra_%.bbappend

```diff
@@ -69,11 +70,17 @@ SRC_URI_append_jetson-nano-2gb-devkit = " \
 # In l4t 28.2 below partitions were 0xC and 0xD
 BALENA_BOOT_PART_jetson-tx2 = "0x18"
 BALENA_DEFAULT_ROOT_PART_jetson-tx2 = "0x19"
+BALENA_BOOT_PART_jetson-tx2-4gb = "0x18"
+BALENA_DEFAULT_ROOT_PART_jetson-tx2-4gb = "0x19"
 
 SRC_URI_append_jetson-tx2 = " \
     file://Add-part-index-command.patch \
     file://tx2-Integrate-with-Balena-u-boot-environment.patch \
 "
+SRC_URI_append_jetson-tx2-4gb = " \
```

Comment:
These ones can be dropped when extending the jetson-tx2.conf machine since it's the same u-boot version and same machine defconfig